udaykalra/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,9 +25,25 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  ArrayList<String> comments;
```

Comment:
Is ArrayList needed, or is the List interface sufficient? In general, exposing less (or more generic) API is preferred, as it can reduce effort needed to refactor code.

Reply:
We were told to use ArrayList specifically.